specification/network/resource-manager/readme.md

```diff
@@ -2552,6 +2552,7 @@ This is not used by Autorest itself.
 
 ``` yaml $(swagger-to-sdk)
 swagger-to-sdk:
+  - repo: azure-sdk-for-net
```

Comment:
why we need track 1 config for dotnet?

Reply:
network has a grace period for track 1 deprecation. We will continue to release track 1 package for network during this period.